modules/claims_api/lib/claims_api/evss_bgs_mapper.rb

```diff
@@ -25,7 +25,10 @@ def map_and_build_object # rubocop:disable Metrics/MethodLength
       claim['data']['development_letter_sent'] = @data['development_letter_sent']
       claim['data']['decision_letter_sent'] = @data['decision_notification_sent']
       claim['data']['documents_needed'] = @data['wwsnfy'].present?
+
       claim['data']['waiver5103_submitted'] = @data['filed5103_waiver_ind'] == 'Y'
+      claim['data']['decision_requested'] = @data['filed5103_waiver_ind'] == 'Y'
```

Comment:
interesting.

Reply:
why do we have 2 values set by the same thing.. that seems odd, but 🤷

Reply:
I checked with Kayla, and that is part of the fun of v1